operators/gpu-operator-certified/v23.9.2/manifests/gpu-operator-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,957 @@
+          - configmaps
+          - secrets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - controllerrevisions
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          - prometheusrules
+          verbs:
+          - get
+          - create
+          - list
+          - update
+          - watch
+          - delete
+        - apiGroups:
+          - apps
+          resourceNames:
+          - gpu-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - nvidia.com
+          resources:
+          - '*'
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - '*'
+        - apiGroups:
+          - policy
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - create
+          - get
+          - update
+          - list
+          - delete
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - list
+          - watch
+      deployments:
+      - name: gpu-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app.kubernetes.io/component: gpu-operator
+              app: gpu-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app.kubernetes.io/component: gpu-operator
+                app: gpu-operator
+                nvidia.com/gpu-driver-upgrade-drain.skip: "true"
+            spec:
+              priorityClassName: system-node-critical
+              containers:
+              - args:
+                  - --leader-elect
+                  - --leader-lease-renew-deadline
+                  - "60s"
+                image: nvcr.io/nvidia/gpu-operator@3988b3b570cf65b0d338cb29bd24b0b9bf1885e1563fbb398ef23cba0b59e6bb
```

Comment:
Hi @cdesiniotis this line is incorrect, it should be

`nvcr.io/nvidia/gpu-operator@sha256:3988b3b570cf65b0d338cb29bd24b0b9bf1885e1563fbb398ef23cba0b59e6bb`

Can you please double check that all of the other image refs are correct as well?

Reply:
Thanks for pointing this out. I have now fixed this and verified the other images.